src/app/admin/manageUser/EditUserForm.tsx

```diff
@@ -21,18 +34,65 @@ interface Props {
 function isValidUserRole(userRole: DisplayedUserRole): userRole is UserRole {
     return userRole !== "UNKNOWN";
 }
+const emailRegex = /^\S+@\S+$/;
+
+const initialFormErrors: InviteUserErrors = {
+    email: Errors.none,
+    role: Errors.none,
+    firstName: Errors.none,
+    lastName: Errors.none,
+    telephoneNumber: Errors.none,
+};
+
+type InviteUserErrors = Required<FormErrors<InviteUserFields>>;
 
 const EditUserForm: React.FC<Props> = (props) => {
     const initialRole: UserRole = isValidUserRole(props.userToEdit.userRole)
         ? props.userToEdit.userRole
         : "volunteer";
 
-    const [role, setRole] = useState<UserRole>(initialRole);
+    const initialFirstName: string =
+        props.userToEdit.firstName !== null ? props.userToEdit.firstName : "";
+
+    const initialLastName: string =
+        props.userToEdit.lastName !== null ? props.userToEdit.lastName : "";
+
+    const initialEmail: string = props.userToEdit.email !== null ? props.userToEdit.email : "";
+
+    const initialPhone: string =
+        props.userToEdit.telephoneNumber !== null ? props.userToEdit.telephoneNumber : "";
```

Comment:
All of these can be simplified with the nullish coalescing operator:
`const initialFirstName: string =  props.userToEdit.firstName ? "";`
Though that's probably terse enough to just write inline in the initial fields

Reply:
Fixed now, the code was complaining last week so didn't adopt this form, but now it's not complaining any more